src/MoBi.UI/Views/ExpressionProfileBuildingBlockView.cs

```diff
@@ -33,12 +33,26 @@ public partial class ExpressionProfileBuildingBlockView : BaseUserControl, IExpr
       private readonly UxComboBoxUnit<ExpressionParameterDTO> _unitControl;
       private readonly ValueOriginBinder<ExpressionParameterDTO> _valueOriginBinder;
 
+      private readonly PopupContainerControl _popupControl = new PopupContainerControl();
+      private readonly RepositoryItemPopupContainerEdit _repositoryItemPopupContainerEdit = new RepositoryItemPopupContainerEdit();
+      private IGridViewBoundColumn<ExpressionParameterDTO, double?> _valueColumn;
+
       public ExpressionProfileBuildingBlockView(ValueOriginBinder<ExpressionParameterDTO> valueOriginBinder)
       {
          InitializeComponent();
          _valueOriginBinder = valueOriginBinder;
          _gridViewBinder = new GridViewBinder<ExpressionParameterDTO>(gridView);
          _unitControl = new UxComboBoxUnit<ExpressionParameterDTO>(gridControl);
+         _repositoryItemPopupContainerEdit.PopupControl = _popupControl;
+         _repositoryItemPopupContainerEdit.QueryDisplayText += (o, e) => OnEvent(queryText, e);
+      }
+
+      private void queryText(QueryDisplayTextEventArgs e)
+      {
+         var distributedParameter = _gridViewBinder.FocusedElement;
+         if (distributedParameter == null)
+            return;
+         e.DisplayText = distributedParameter.ExpressionParameterFormatter().Format(distributedParameter.Value);
```

Comment:
This is the line responsible for the formatting when the popup is being shown. With this line, it does not change when it's focused, unfocused, or when the popup is showing.